### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by inad9300**

### CODE REVIEW

File: releases/22.0.0.12/kernel-slim/helpers/runtime/docker-server.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ function importKeyCert() {
   local KEYSTORE_FILE="/output/resources/security/key.p12"
   local TRUSTSTORE_FILE="/output/resources/security/trust.p12"
 
-  # Import the private key and certificate into new keytore
+  # Import the private key and certificate into new keystore
```
Comment (leochr) [inline releases/22.0.0.12/kernel-slim/helpers/runtime/docker-server.sh:16]: @inad9300 Thanks for the PR. Could you please update all the other instances of `keytore` in other Dockerfiles?

Comment (inad9300) [reply releases/22.0.0.12/kernel-slim/helpers/runtime/docker-server.sh:16]: Done.

Comment (leochr) [reply releases/22.0.0.12/kernel-slim/helpers/runtime/docker-server.sh:16]: @inad9300 Thanks for the quick update. Could you please rebase? images for 22.0.0.13 are no longer maintained, instead, 23.0.0.1 should be updated with the fix.

Comment (inad9300) [reply releases/22.0.0.12/kernel-slim/helpers/runtime/docker-server.sh:16]: Done.

**Review by leochr — APPROVED** [review]
@inad9300 Looks good. Thank you

**Pull request merged**